packages/atomic/src/components/atomic-result/atomic-result.tsx

```diff
@@ -68,13 +71,25 @@ export class AtomicResult {
    */
   @Prop() image: ResultDisplayImageSize = 'icon';
 
+  @Prop() classes = '';
+
   @Listen('atomic/resolveResult')
   public resolveResult(event: ResultContextEvent<FoldedResult | Result>) {
     event.preventDefault();
     event.stopPropagation();
     event.detail(this.result);
   }
 
+  @Listen('atomic/resolveResultDisplayConfig')
+  public resolveResultDisplayConfig(event: ResultContextEvent<DisplayConfig>) {
+    event.preventDefault();
```

Comment:
I think it's better to leave it bc it returns the closest parent - if we let it bubble up it could get the data from the grandparent atomic-result (which is not the end of the world, but I think this makes more sense)